### PR TITLE
CSP Directives

### PR DESCRIPTION
**Pull request by paul-vd**
### Implementation
- https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2033

### Issue
- https://app.intercom.com/a/inbox/ehgzoeah/inbox/admin/5309138/conversation/188350900000787?view=List

### CODE REVIEW

File: versioned_docs/version-2.x/appendices/migration-guides/index.mdx [hunk 1]
```diff
@@ -49,7 +63,8 @@ the following files:
 
 ### New features in `2.23.0`
 
-- [Use Magento's `storeConfig` GraphQL query](/docs/2.x/magento2/using-magento-configuration#use-magentos-storeconfig-graphql-query) to access configurations with the `MagentoGraphQLConfig` loader
+- [Use Magento's `storeConfig` GraphQL query](/docs/2.x/magento2/using-magento-configuration#use-magentos-storeconfig-graphql-query)
+  to access configurations with the `MagentoGraphQLConfig` loader
```
Comment (paul-vd) [inline versioned_docs/version-2.x/appendices/migration-guides/index.mdx:67]: prettier change 😭

File: versioned_docs/version-2.x/reference/content-security-policy.mdx [hunk 2]
```diff
@@ -20,13 +20,19 @@ each:
...
     directives: {
-      scriptSrc: [],
-      frameSrc: [],
-      styleSrc: [],
-      imgSrc: [],
-      fontSrc: [],
-      connectSrc: [],
-      baseUri: []
+        defaultSrc: [],
+        scriptSrc: [],
```
Comment (dpobel) [inline versioned_docs/version-2.x/reference/content-security-policy.mdx:24]: why changing the indent level ?

File: docs/api-reference/front-commerce-remix.mdx
Comment (dpobel) [inline docs/api-reference/front-commerce-remix.mdx:1]: this should be done in a separate MR. I guess this is a fix for the build failure I have in https://github.com/front-commerce/developers.front-commerce.com/pull/662 (that way 662 could be based on it)

File: versioned_docs/version-2.x/appendices/migration-guides/index.mdx [hunk 3]
```diff
@@ -12,6 +12,20 @@ import ContactLink from "@site/src/components/ContactLink";
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## `2.23.0` -> `2.24.0`
+
+### CSP directives
```
Comment (dpobel) [inline versioned_docs/version-2.x/appendices/migration-guides/index.mdx:17]: strictly speaking, while migrating from 2.23.0 to 2.24.0 there's nothing to change in that regard. That said, in previous releases we took the habit to have a "New features" paragraph, I think this is where you could advertise the new CSP directives.

**Review by real34 — APPROVED** [review]

**Pull request merged**